app/components/candidate_interface/other_qualifications_review_component.rb

```diff
@@ -35,8 +35,8 @@ def other_qualifications_rows(qualification)
     def no_qualification_row
       params = { change: true }.merge(return_to_params)
       [{
-        key: 'Do you want to add any A levels and other qualifications',
-        value: 'No',
+        key: 'A levels and other qualifications',
+        value: 'None added',
```

Comment:
Should we use the actual answer from the radio label maybe?

```suggestion
        value: 'I do not want to add any A levels and other qualifications',
```

That might make more sense alongside the "Change" link. On the other hand, it's more long-winded and possibly "None added" is clearer? 🤔